Lib/idlelib/editor.py

```diff
@@ -86,10 +87,19 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
                     dochome = os.path.join(basepath, pyver,
                                            'Doc', 'index.html')
             elif sys.platform[:3] == 'win':
-                chmfile = os.path.join(sys.base_prefix, 'Doc',
-                                       'Python%s.chm' % _sphinx_version())
-                if os.path.isfile(chmfile):
-                    dochome = chmfile
+                docfile = "Dummy^file&name*that(should)fail-isfile#check"
```

Comment:
Could also just set it to `None` and check that later?

Reply:
Done.  Looks better.

Reply:
Yes.  "" seems even better since isfile("") is False, at least on Win10.  True on all Windows?

Reply:
I believe it's always worked like that - I don't see how it'll ever be an actual file, which means the FileNotFoundError check will make it return False.

Reply:
Then do you approve of this, and in particular, for backporting to 3.11 *now*, so that Windows 3.11.0 users can access the new offline html doc?

Reply:
Yep